project_future.py

```diff
@@ -584,6 +584,9 @@ def add_arguments(parser):
     parser.add_argument("--report-time-path",
                         help='export time for building each xcode build target to the specified json file',
                         type=os.path.abspath)
+    parser.add_argument("--clangc",
```

Comment:
Does it make sense to name this argument `clangc` instead of `clang`? With the current naming, I expect I should pass in a `clangc` executable (which doesn’t exist AFAICT). 

Note that for Swift we actually have two different symlinks to the compiler: `swift` which can also do things like `swift build` and `swiftc` which is the compiler itself, so there’s a difference.

Reply:
I updated to use `clang` instead of `clangc`.